templates/ehub-existing-setup.json

```diff
@@ -0,0 +1,267 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "Name": {
+            "type": "String"
+        },
+        "Storage Name": {
+            "type": "String"
+        },
+        "Alert Logic Access Key ID": {
+            "type": "String"
+        },
+        "Alert Logic Secret Key": {
+            "type": "SecureString"
+        },
+        "Alert Logic API endpoint": {
+            "type": "String",
+            "defaultValue": "api.global-services.global.alertlogic.com",
+            "allowedValues" : [
+                "api.global-services.global.alertlogic.com",
+                "api.global-integration.product.dev.alertlogic.com"
+            ]
+        },
+        "Alert Logic Data Residency": {
+            "type": "String",
+            "defaultValue": "default",
+            "allowedValues": [
+                "default"
+            ]
+        },
+        "Service Principal ID": {
+            "type": "String"
+        },
+        "App Client ID": {
+            "type": "String"
+        },
+        "App Client Secret": {
+            "type": "SecureString"
+        },
+        "Event Hub Resource Group": {
+            "type": "String"
+        },
+        "Event Hub Namespace Name": {
+            "type": "String"
+        },
+        "Event Hub Name": {
+            "type": "String"
+        },
+        "Event Hub Connection String": {
+            "type": "SecureString"
+        },
+        "Event Hub Consumer Group": {
+            "type": "String"
+        }
+    },
+    "variables": {
+        "subscriptionId": "[subscription().subscriptionId]",
+        "location": "[resourceGroup().location]",
+        "resourceGroupName": "[resourceGroup().name]",
+        "deploymentName" : "[deployment().name]",
+        "resourceGroupId": "[resourceGroup().id]",
+        "roleAssignmentId": "[guid(uniqueString( resourceGroup().id, variables('deploymentName')))]",
+        "targetEhubResourceGroupId": "[concat('/subscriptions/', variables('subscriptionId'), '/resourceGroups/', parameters('Event Hub Resource Group'))]",
+        "targetRoleAssignmentId": "[guid(uniqueString(variables('targetEhubResourceGroupId'), variables('deploymentName')))]",
```

Comment:
Seems not to be used.
(And if not needed, the previous one - targetEhubResourceGroupId wouldn't be used, either.)